source/_integrations/sensibo.markdown

```diff
@@ -249,3 +391,63 @@ switch:
 ```
 
 {% endraw %}
+
+### Start the timer for 30 minutes when I get home
+
+{% raw %}
+
+```yaml
```

Comment:
These YAML examples do not follow our YAML-styling guides. Please adjust them to comply with:

https://developers.home-assistant.io/docs/documenting/yaml-style-guide

Additionally, please leave out defaults from these examples.